### PR TITLE
Update website to latest dependencies.

### PR DESCRIPTION
**Pull request by mattsoulanille**
This change is [Reviewable](https://reviewable.io/reviews/tensorflow/tfjs-website/462)

### CODE REVIEW

File: source/_data/api/3.19.0/skeleton.json [hunk 1]
```diff
@@ -0,0 +1,314 @@
...
+      }
+    ]
+  },
+  {
+    "name": "Models",
+    "description": [
+      "<p>Models are one of the primary abstractions used in",
+      "TensorFlow.js Layers.  Models can be trained, evaluated, and used",
+      "for prediction.  A model's state (topology, and optionally, trained",
+      "weights) can be restored from various formats.</p>",
```
Comment (jasonmayes) [inline source/_data/api/3.19.0/skeleton.json:65]: Would be good to detail formats supported?

File: source/_data/api/3.19.0/skeleton.json [hunk 2]
```diff
@@ -0,0 +1,314 @@
...
+          "time"
+        ]
+      }
+    ]
+  },
+  {
+    "name": "Environment",
+    "description": [
+      "<p>TensorFlow.js can run mathematical operations on",
+      "different backends. Currently, we support WebGL and JavaScript",
```
Comment (jasonmayes) [inline source/_data/api/3.19.0/skeleton.json:249]: Should we not mention WASM and that WebGPU is in alpha?

**Review by jasonmayes — APPROVED** [review]
LGTM - added a few comments but realised you were just updating the version numbers I am guessing. Looks like when we get a doc writer we should update some of this stuff but  otherwise LGTM.

**Review by jasonmayes — APPROVED** [review]

**Review by pyu10055 — APPROVED** [review]
> Reviewed 20 of 20 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/tensorflow/tfjs-website/462)* status: :shipit: complete! 1 of 1 approvals obtained (waiting on @mattsoulanille)

**Pull request merged**